src/views/Project/index.jsx

```diff
@@ -158,6 +161,23 @@ export default class Project extends Component {
       this.load();
     }
   }
+
+  handleBugInfoClick = async id => {
+    try {
+      const {
+        data: { comments },
+      } = await this.props.client.query({
+        query: commentsQuery,
+        variables: { id },
+        context: { link: 'bugzilla' },
```

Comment:
Where is the `context` used?

Reply:
context is to choose between bugzilla / github link afaik

Reply:
Could you point me to where `context` is being used for the comment?

Reply:
Could you point me to where this is being used for the comment?

Reply:
https://github.com/mozilla-frontend-infra/codetribute/blob/master/src/App.jsx#L35 we use the split method which will choose what link to send request to according to the context, 

but now that i see the condition, since we check for github otherwise set it to bugzilla, maybe we can remove it in here?

Reply:
Oh, shouldn't this be `context: { client: 'bugzilla' },` instead?

Reply:
OHHH so thats what u mean!! 🤦‍♀️